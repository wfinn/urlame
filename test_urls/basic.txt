http://example.com/lame.png?x=y
http://example.com/docs/boring
https://example.org/user/rick
https://example.org/user/morty

https://example.com/trailingslash
http://example.com/trailingslash/?

http://example.org/foo/uselessparam/
http://example.org/foo/uselessparam/?utm_source=twitter

http://example.com/en_US/langcode
http://example.com/de/langcode
http://example.com/de_DE/langcode

http://example.com/numbers/year-2018-pix
http://example.com/numbers/year-2017-pix
http://example.com/numbers/year-2016-pix

https://localhost/files/123.json
https://localhost/files/456.json
https://localhost/files/admin.json

https://example.org/params?A=something
https://example.org/params?A=something_else
https://example.org/params?A=something_else_entirely
https://example.org/params?B=something

https://example.com/?encoded[1]=xyz
https://example.com/?encoded%5B1%5D=xyz

https://example.com/en-US/combined/item-1
https://example.com/de-DE/combined/item-5
https://example.com/de-DE/combined/item-deletion
