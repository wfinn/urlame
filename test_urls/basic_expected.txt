https://example.com/trailingslash
http://example.org/foo/uselessparam/
http://example.com/en_US/langcode
http://example.com/numbers/year-2018-pix
https://localhost/files/123.json
https://localhost/files/admin.json
https://example.org/params?A=something
https://example.org/params?B=something
https://example.com/?encoded[1]=xyz
https://example.com/en-US/combined/item-1
https://example.com/de-DE/combined/item-deletion
https://example.com/
https://example.com/robots.txt?newparam=1
http://newsubdomain.example.com/robots.txt
https://example.com/newdir/robots.txt
https://example.com/robots.txt/itsadir
